x/distribution/client/cli/cli_test.go

```diff
@@ -309,7 +309,7 @@ func (s *IntegrationTestSuite) TestGetCmdQueryValidatorSlashes() {
 				sdk.ValAddress(val.Address).String(), "1", "3",
 			},
 			false,
-			"null",
+			"pagination: {}\nslashes: null",
```

Comment:
I didn't see any failing test, but shouldn't there be a `,` here?

Reply:
this isn't a json output, I think there won't be any `,` for text output

Reply:
ah yeah you're right, I got it mixed up 👍